cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -90,19 +88,23 @@ data ForkPoint
 startTimeOffsetSeconds :: DTC.NominalDiffTime
 startTimeOffsetSeconds = if OS.isWin32 then 90 else 15
 
--- | Like 'cardanoTestnet', but using defaults for all configuration files.
+-- | Like 'cardanoTestnet', but using 'ShelleyTestnetOptions' to obtain
+-- the genesis files, instead of passing them directly.
 -- See 'cardanoTestnet' for additional documentation.
 cardanoTestnetDefault
   :: ()
   => HasCallStack
   => CardanoTestnetOptions
+  -> ShelleyTestnetOptions
```

Comment:
We now have two testnet options types here. This is not ideal.

Reply:
Maybe the naming is bad. Would `ShelleyTestnetOptions` better be named `GenesisOptions` maybe?

Reply:
`GenesisOptions` is better.

Reply:
PR doing the renaming: https://github.com/IntersectMBO/cardano-node/pull/5987